src/hooks/protections/received_event.cpp

```diff
@@ -582,23 +582,26 @@ namespace big
 
 			if (g_local_player && g_local_player->m_net_object && g_local_player->m_net_object->m_object_id == net_id)
 			{
+				weapon_item weapon = g_gta_data_service->weapon_by_hash(hash);
 				g_notification_service->push_warning("PROTECTIONS"_T.data(),
-				    std::vformat("REMOVE_WEAPON_ATTEMPT"_T, std::make_format_args(source_player->get_name())));
+					std::format("{} {} {}.", source_player->get_name(), "REMOVE_WEAPON_ATTEMPT"_T, weapon.m_display_name));

```

Comment:
This change will fuck over every other language that had this key translated.

Reply:
What should the remedy be? Change the key?

Reply:
Yeah as it will just be incorrect in other languages since the format doesn't work.